blobfile/_gcp.py

```diff
@@ -150,6 +152,13 @@ def _create_access_token_request(creds: Dict[str, Any], scopes: List[str]) -> Re
 # Derived from:
 # - https://github.com/googleapis/google-auth-library-python/blob/0afc61a3a9c3d5035c913ad4a7568e8a888e2ec9/google/auth/external_account.py#L361
 # - https://github.com/googleapis/google-auth-library-python/blob/0afc61a3a9c3d5035c913ad4a7568e8a888e2ec9/google/oauth2/sts.py#L95
+
+
+@retry(
+    retry=retry_if_exception_type(ValueError),
+    stop=stop_after_attempt(1),
+    wait=wait_fixed(3) + wait_random(0, 2),
+)

```

Comment:
IMHO we should:
- try more than twice
- back off expontentially e.g. something like: `wait_random_exponential(multiplier=1, max=60)` (cribbed from [here](https://tenacity.readthedocs.io/en/latest/#:~:text=When%20multiple%20processes%20are%20in%20contention%20for%20a%20shared%20resource%2C%20exponentially%20increasing%20jitter%20helps%20minimise%20collisions.))

Reply:
took this from our existing approach that @l1n suggested - happy to retry more. how about trying ~3 times with exponential backoff?

Reply:
and yeah, it's one retry for a max of two attempts.

Reply:
honestly either way sgtm

Reply:
i was :sus: about 1 retry, so more sgtm. adding in, will test manually.